test/AWS.Deploy.CLI.IntegrationTests/WebAppNoDockerFileTests.cs

```diff
@@ -107,6 +108,9 @@ public async Task DefaultConfigurations()
             Assert.True(Directory.Exists(Path.Combine(_customWorkspace, "temp")));
             Assert.True(Directory.Exists(Path.Combine(_customWorkspace, "Projects")));
 
+            // Verify existence of the saved deployment settings file
+            Assert.True(File.Exists(saveSettingsFilePath));
```

Comment:
We aren't testing if the file can actually be used for deployment.

Reply:
Modified an existing integ test to use the config file created via `_deploymentSettingsHandler.SaveSettings(...)` method